model/categoryFilter.js

```
@@ -0,0 +1,41 @@
+import { action, computed, decorate, observable } from 'mobx';
+
+export class CategoryFilter {
+	constructor(filterId, title, filter) {
+		this.id = filterId;
+		this.title = title;
+		this.filter = filter;
+		this.selectedCategories = new Set();
+	}
+
+	get isApplied() {
+		return this.selectedCategories.size > 0;
+	}
+
+	set isApplied(isApplied) {
```

So, `filter.IsApplied = true` does nothing, right? 
Should we throw an error in this case to help diagnose wrong usage?

---

I considered it, but I wouldn't want to break the page for that. It would be great if we could easily send a warning that goes in (for example) the LMS log.